merkle/smt/tile.go

```diff
@@ -28,3 +28,34 @@ type Tile struct {
 	ID     tree.NodeID2
 	Leaves []Node
 }
+
+// scan visits all non-empty nodes of the tile except the root.
+func (t Tile) scan(lo *tree.Layout, h mapHasher, visit func(tree.NodeID2, []byte)) error {
```

Comment:
nitty: "Layout" feels like one word to me, so 'l'?

Reply:
Done.